orbit/pkg/useraction/mdm_migration_darwin.go

```diff
@@ -181,9 +181,10 @@ func NewMDMMigrator(path string, frequency time.Duration, handler MDMMigratorHan
 // the underlying mechanism for user action.
 type swiftDialogMDMMigrator struct {
 	*baseDialog
-	props     MDMMigratorProps
-	frequency time.Duration
-	handler   MDMMigratorHandler
+	props       MDMMigratorProps
+	frequency   time.Duration
+	frequencyMu sync.RWMutex
```

Comment:
Hey, I was looking at where this is being read in the code, and it looks like it is just in `ShowInterval`, which is protected by the `lastShownMu`. Seems like we can just use that mutex for both?

Reply:
Nice! Would it be worth having its own just in case `frequency` ends up needing to be used anywhere else?

Reply:
I think because it is so closely tied to the lastShown timestamp that we'll probably always want to be locking them together. Plus that makes it easier than having two locks that we need to try when accessing it in what we think will be the common scenario. If we come across a scenario where that isn't true, we can always add a second mutex later.